components/faunadb/actions/execute-graphql-query/execute-graphql-query.mjs

```diff
@@ -0,0 +1,24 @@
+import faunadb from "../../faunadb.app.mjs";
+
+export default {
+  key: "faunadb-execute-graphql-query",
+  name: "Execute GraphQL Query",
+  description: "Performs an arbitrary authorized GraphQL query",
```

Comment:
All actions should include a link to the relevant documentation in the description.